From acb9be2eaf9321c70beae16cb31b51a226893f92 Mon Sep 17 00:00:00 2001
From: Doug Ledford <dledford@redhat.com>
Date: Wed, 24 Sep 2014 17:27:59 -0400
Subject: [net] svcrdma: fix printk when memory allocation fails

Message-id: <5214b1d06fdcef8249ea9b1bf84a5035e5753fc6.1411578494.git.dledford@redhat.com>
Patchwork-id: 96658
O-Subject: [Patch RHEL7 06/26] svcrdma: fix printk when memory allocation fails
Bugzilla: 1107621
RH-Acked-by: Jes Sorensen <Jes.Sorensen@redhat.com>
RH-Acked-by: Steve Dickson <SteveD@redhat.com>

From: Jeff Layton <jlayton@redhat.com>

It retries in 1s, not 1000 jiffies.

Signed-off-by: Jeff Layton <jlayton@redhat.com>
Signed-off-by: J. Bruce Fields <bfields@redhat.com>
(cherry picked from commit c42a01eee74dfd9ba8f8abb7cb81dd9a8839dc7b)
Signed-off-by: Doug Ledford <dledford@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/net/sunrpc/xprtrdma/svc_rdma_transport.c b/net/sunrpc/xprtrdma/svc_rdma_transport.c
index 941b140..54a232e 100644
--- a/net/sunrpc/xprtrdma/svc_rdma_transport.c
+++ b/net/sunrpc/xprtrdma/svc_rdma_transport.c
@@ -480,8 +480,7 @@ struct page *svc_rdma_get_page(void)
 
  while ((page = alloc_page(GFP_KERNEL)) == NULL) {
   /* If we can't get memory, wait a bit and try again */
-  printk(KERN_INFO "svcrdma: out of memory...retrying in 1000 "
-         "jiffies.\n");
+  printk(KERN_INFO "svcrdma: out of memory...retrying in 1s\n");
   schedule_timeout_uninterruptible(msecs_to_jiffies(1000));
  }
  return page;
-- 
1.7.1